packages/flutter_tools/test/commands.shard/hermetic/doctor_test.dart

```diff
@@ -524,30 +535,34 @@ void main() {
         }
       }
     }, overrides: <Type, Generator>{
+      Artifacts: () => mockArtifacts,
+      FileSystem: () => memoryFileSystem,
       OutputPreferences: () => OutputPreferences(wrapText: false),
       ProcessManager: () => mockProcessManager,
       Platform: _kNoColorOutputPlatform,
     });
 
     testUsingContext('gen_snapshot binary not available', () async {
-        expect(await FlutterValidatorDoctor().diagnose(verbose: false), isTrue);
-        // gen_snapshot is downloaded on demand, and the doctor should not
-        // fail if the gen_snapshot binary is not present.
-        expect(testLogger.statusText, contains('No issues found!'));
+      expect(await FlutterValidatorDoctor().diagnose(verbose: false), isTrue);
+      // gen_snapshot is downloaded on demand, and the doctor should not
```

Comment:
Is this true?

Reply:
I guess? For context https://github.com/flutter/flutter/pull/52574.